yt/chyt/controller/internal/strawberry/oplet.go

```diff
@@ -362,6 +367,19 @@ func (oplet *Oplet) CheckOperationLiveness(ctx context.Context) error {
 		log.String("operation_state", string(ytOp.State)))
 
 	oplet.persistentState.YTOpState = ytOp.State
+
+	// oplet.persistentState.YTOpSuspended = ytOp.Suspend
```

Comment:
The comment below explains why :)
Actually, since https://github.com/ytsaurus/ytsaurus/commit/f3d2c695fb7ad70cb690b25f782a8f83980490ed has been merged, maybe it's time to uncomment the line - I'll check this

Reply:
There was no go sdk release since the merge, so this crutch is still necessary :(